.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: "Build Packages"
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  macos:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - uses: cachix/install-nix-action@v10
+      with:
+        nix_path: nixpkgs=.
+    - uses: kloenk/ofbuild@0.0.2
```

Comment:
I think it makes a lot of sense to put the action into nixpkgs rather than being external.

Reply:
Makes sense the code itself just parses git commit messages so complexity will be kept low.

Reply:
The complete source code, or just the compile is asset?

Also not exactly sure if she need a action.yml if we move it into nixpkgs

Reply:
If so the complete code.